x-pack/plugins/observability_solution/slo/public/hooks/use_fetch_historical_summary.ts

```diff
@@ -21,16 +21,28 @@ export interface UseFetchHistoricalSummaryResponse {
 }
 
 export interface Params {
-  list: Array<{ sloId: string; instanceId: string }>;
+  sloList: SLOWithSummaryResponse[]; // TODO Kevin: NOT THE CORRECT TYPE
   shouldRefetch?: boolean;
 }
 
 export function useFetchHistoricalSummary({
-  list = [],
+  sloList = [],
   shouldRefetch,
 }: Params): UseFetchHistoricalSummaryResponse {
   const { http } = useKibana().services;
 
+  // TODO Kevin: Check if we want to have an API using partial SLO model
```

Comment:
Is this still TODO?

Reply:
Yes, but probably won't do it in this PR. Should i create an issue and remove the TODO?